{"metadata":{"navigatorTitle":[{"kind":"identifier","text":"NetworkError"}],"fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"NetworkError","kind":"identifier"}],"roleHeading":"Enumeration","externalID":"s:16CryptoTrackerApp14NetworkManagerC0D5ErrorO","role":"symbol","symbolKind":"enum","modules":[{"name":"CryptoTrackerApp"}],"title":"NetworkManager.NetworkError"},"topicSections":[{"title":"Enumeration Cases","identifiers":["doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError\/badURLResponse(url:)","doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError\/unkown"]},{"title":"Instance Properties","identifiers":["doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError\/errorDescription"]},{"title":"Default Implementations","identifiers":["doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError\/Error-Implementations","doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError\/LocalizedError-Implementations"],"generated":true}],"sections":[],"schemaVersion":{"major":0,"minor":3,"patch":0},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp","doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager"]]},"relationshipsSections":[{"title":"Conforms To","identifiers":["doc:\/\/Currency.CryptoTrackerApp\/10Foundation14LocalizedErrorP","doc:\/\/Currency.CryptoTrackerApp\/s5ErrorP","doc:\/\/Currency.CryptoTrackerApp\/s8SendableP"],"kind":"relationships","type":"conformsTo"}],"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"tokens":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"NetworkError","kind":"identifier"}],"languages":["swift"]}],"kind":"declarations"}],"variants":[{"paths":["\/documentation\/cryptotrackerapp\/networkmanager\/networkerror"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError","interfaceLanguage":"swift"},"references":{"doc://Currency.CryptoTrackerApp/documentation/CryptoTrackerApp/NetworkManager/NetworkError/Error-Implementations":{"abstract":[],"url":"\/documentation\/cryptotrackerapp\/networkmanager\/networkerror\/error-implementations","type":"topic","identifier":"doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError\/Error-Implementations","kind":"article","role":"collectionGroup","title":"Error Implementations"},"doc://Currency.CryptoTrackerApp/documentation/CryptoTrackerApp/NetworkManager/NetworkError/unkown":{"url":"\/documentation\/cryptotrackerapp\/networkmanager\/networkerror\/unkown","abstract":[],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unkown","kind":"identifier"}],"type":"topic","identifier":"doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError\/unkown","kind":"symbol","role":"symbol","title":"NetworkManager.NetworkError.unkown"},"doc://Currency.CryptoTrackerApp/s5ErrorP":{"type":"unresolvable","identifier":"doc:\/\/Currency.CryptoTrackerApp\/s5ErrorP","title":"Swift.Error"},"doc://Currency.CryptoTrackerApp/s8SendableP":{"type":"unresolvable","identifier":"doc:\/\/Currency.CryptoTrackerApp\/s8SendableP","title":"Swift.Sendable"},"doc://Currency.CryptoTrackerApp/10Foundation14LocalizedErrorP":{"type":"unresolvable","identifier":"doc:\/\/Currency.CryptoTrackerApp\/10Foundation14LocalizedErrorP","title":"Foundation.LocalizedError"},"doc://Currency.CryptoTrackerApp/documentation/CryptoTrackerApp/NetworkManager/NetworkError/errorDescription":{"url":"\/documentation\/cryptotrackerapp\/networkmanager\/networkerror\/errordescription","abstract":[],"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"type":"topic","identifier":"doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError\/errorDescription","kind":"symbol","role":"symbol","title":"errorDescription"},"doc://Currency.CryptoTrackerApp/documentation/CryptoTrackerApp":{"identifier":"doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp","abstract":[],"type":"topic","role":"collection","url":"\/documentation\/cryptotrackerapp","kind":"symbol","title":"CryptoTrackerApp"},"doc://Currency.CryptoTrackerApp/documentation/CryptoTrackerApp/NetworkManager/NetworkError/LocalizedError-Implementations":{"abstract":[],"type":"topic","identifier":"doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError\/LocalizedError-Implementations","kind":"article","role":"collectionGroup","url":"\/documentation\/cryptotrackerapp\/networkmanager\/networkerror\/localizederror-implementations","title":"LocalizedError Implementations"},"doc://Currency.CryptoTrackerApp/documentation/CryptoTrackerApp/NetworkManager/NetworkError":{"type":"topic","abstract":[],"identifier":"doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError","kind":"symbol","navigatorTitle":[{"text":"NetworkError","kind":"identifier"}],"role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"NetworkError"}],"url":"\/documentation\/cryptotrackerapp\/networkmanager\/networkerror","title":"NetworkManager.NetworkError"},"doc://Currency.CryptoTrackerApp/documentation/CryptoTrackerApp/NetworkManager":{"type":"topic","abstract":[],"identifier":"doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"NetworkManager"}],"role":"symbol","fragments":[{"text":"class","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"NetworkManager"}],"url":"\/documentation\/cryptotrackerapp\/networkmanager","title":"NetworkManager"},"doc://Currency.CryptoTrackerApp/documentation/CryptoTrackerApp/NetworkManager/NetworkError/badURLResponse(url:)":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"badURLResponse","kind":"identifier"},{"kind":"text","text":"("},{"text":"url","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:10Foundation3URLV","text":"URL","kind":"typeIdentifier"},{"text":")","kind":"text"}],"title":"NetworkManager.NetworkError.badURLResponse(url:)","type":"topic","abstract":[],"identifier":"doc:\/\/Currency.CryptoTrackerApp\/documentation\/CryptoTrackerApp\/NetworkManager\/NetworkError\/badURLResponse(url:)","url":"\/documentation\/cryptotrackerapp\/networkmanager\/networkerror\/badurlresponse(url:)","role":"symbol","kind":"symbol"}}}